client/src/state/H5PEditor/H5PEditorActions.ts

```diff
@@ -130,7 +130,11 @@ export function exportH5P(includeReporter: boolean): any {
             try {
                 await api.exportAsHtml(data.contentId, includeReporter);
 
-                track('H5P', 'export', 'tracker', `${includeReporter}`);
+                track(
+                    'H5P',
+                    'export_as_html',
```

Comment:
Maybe we could already add single_html to differentiate from the planned export with external resources